### PR TITLE
Bump 4.0.0.pre8

### PR DESCRIPTION
**Pull request by jasonrclark**
Used the [Release Checklist on wiki](https://github.com/shoes/shoes4/wiki/Release-Checklist) that I'd forgotten exist, and have updated!

* [x] [Merge outstanding PRs](https://github.com/shoes/shoes4/pulls)
* [x] Increment version number (edit VERSION and run `rake update_version` to update numbers)
* [x] Run through the samples on all platforms
  * [x] Mac
  * [x] Windows
  * [x] Linux
* [x] Run packaging on all platforms from installed `shoes` command. Ensure jars run on all platforms, app on Mac (currently)
  * [x] Mac
  * [x] Windows
  * [x] Linux
* [x] Write release Blog post (https://github.com/shoes/shoesrb.com/pull/80)
* [x] Merge this
* [ ] Release the shoooooooes! (`rake release:all`)
* [ ] Send message to mailing list
* [ ] Release blog post
* [ ] :tada: :bell: :triumph:

## For the _Next_ Release
* [ ] Once finished celebrating, look for gem updates!
* [ ] Check out #1372... should be ready to fix

### CODE REVIEW

**Comment by PragTob** [comment]
Was about to ask you when we wanted to launch the next release :)

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,33 @@
...
+Improvements (7)
+-----------------
+
+* Update to latest SWT version 4.6.1 [9b239e6]
+* New download supporting all options from Shoes 3 [5f51959]
+* Make art element parameters compatible with Shoes 3 [4f0d57f]
+* Apply rotate consistently to all art elements [07cbfba]
+* Better native app appearance on Mac OS X [7af177c][429c363]
+* Warn on second Shoes.app invocation [fb8f44c]
+* Update to latest SWT version 4.6.1 [7ba14db]
```
Comment (PragTob) [inline CHANGELOG]: this one is double!

File: CHANGELOG [hunk 2]
```diff
@@ -1,3 +1,33 @@
...
+
+Bug Fixes (5)
+-------------
+
+* Make rotate direction compatible with Shoes 3 [2781622]
+* Allow chaining click and release methods [33c6695]
```
Comment (PragTob) [inline CHANGELOG]: Huh, I'd put that in as a feature but it's not that important :D

Comment (jasonrclark) [inline CHANGELOG]: Yeah, it's on the border but I moved it up since I was editing it.

File: CHANGELOG (hunk 1 above)
Comment (jasonrclark) [inline CHANGELOG]: 🔪

**Comment by jasonrclark** [comment]
So I actually "cheated" and made a fake branch that I merged all the outstanding work into before generating the `CHANGELOG`. 😁 

Might be a slight change in commit count, but otherwise should be 👍

**Comment by PragTob** [comment]
> So I actually "cheated" and made a fake branch that I merged all the outstanding work into before generating the CHANGELOG. :grin:

[Liar!](https://www.youtube.com/watch?v=oU7rqB9E_0M)

**Comment by jasonrclark** [comment]
Done! 👏

**Comment by PragTob** [comment]
Thanks a bunch jason! :tada: 

Actually wanted to get in and at least do the Linux testing, but you beat me to it :)

**Pull request merged**